Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py

```diff
@@ -0,0 +1,175 @@
+# Install pygame library
+from pygame import gfxdraw
+
```

Comment:
Remove this extra line

Reply:
But this is not extra line

Reply:
This is required for installation of pygame

Reply:
Remove blank line no. 3